src/device-registry/utils/generate-filter.js

```diff
@@ -565,7 +565,7 @@ const generateFilter = {
       }
 
       if (device_id) {
```

Comment:
updating the generator filter logic to cater for a current storage mechanism in the datawarehouse